### DISCUSSION TITLE
Deploy locally trained BERTopic model in Amazon Sagemaker

### DISCUSSION THREAD

**Discussion by Soumadip-Saha** [MaartenGr/BERTopic#738] [Q&A]
Hello everyone. I am new to **Amazon Sagemaker** and have built a locally trained BERTopic model that I would like to deploy in Amazon Sagemaker. I don't know how to convert the saved model into the ".tar.gz" format for deployment. If anyone has any idea how to proceed, please help me with this. Thanks a lot in advance.

**Comment by MaartenGr** [comment]
Although I am not familiar with the deployment of BERTopic in Sagemaker, I believe you can find more information about creating a ".tar.gz" format [here](https://discuss.huggingface.co/t/how-to-create-model-in-sagemaker-console-from-tar-gz/14830).

**Comment by dhea1323** [comment]
@Soumadip-Saha Hello, have you successfully deployed BERTopic in AWS? Can I contact you?
